backend/codesirius/problems/serializers/problem.py

```
@@ -0,0 +1,113 @@
+        queryset=Language.objects.all(),
+        required=True,
+        error_messages={
+            "required": "Minimum one language is required.",
+            "null": "Minimum one language is required.",
+        },
+    )
+    status = serializers.ChoiceField(
+        choices=Problem.Status.choices, default=Problem.Status.DRAFT
+    )
+
+    def validate_tags(self, tags):
```

Ruff (0.8.2)

Missing return type annotation for public function `validate_tags`

(ANN201)